### PR TITLE
chore: use primitive u128 for genesis nonce

### PR DESCRIPTION
**Pull request by quake**

### CODE REVIEW

File: spec/src/lib.rs [hunk 1]
```diff
@@ -99,7 +99,7 @@ pub struct Genesis {
     pub compact_target: u32,
     pub uncles_hash: H256,
     pub hash: Option<H256>,
-    pub nonce: U128,
+    pub nonce: u128,
```
Comment (keroro520) [inline spec/src/lib.rs]: https://github.com/nervosnetwork/ckb/blob/develop/resource/specs/dev.toml#L9

Should `"0x0"` -> `0` ? And integration.toml as well.

Comment (quake) [reply spec/src/lib.rs]: I didn't notice that. now fixed, thanks.

Comment (driftluo) [reply spec/src/lib.rs]: `toml-rs` is not supported parse u128 now

Comment (quake) [reply spec/src/lib.rs]: WTH...

**Pull request closed without merging**